api/lib/domain/models/Session.js

```diff
@@ -15,6 +23,25 @@ const statuses = {
 const NO_EXAMINER_GLOBAL_COMMENT = null;
 
 class Session {
```

Comment:
Quel est le problème que résout le pattern RORO ? Pour pouvoir déstructurer correctement quand il n'y a pas tous les champs fournis
- https://www.freecodecamp.org/news/elegant-patterns-in-modern-javascript-roro-be01e7669cbd/

Reply:
Pas seulement pour ne passer qu'une partie des paramètres, mais aussi (et sur Pix, surtout, j'ai l'impression) pour nommer les paramètres à l'appel. Personnellement au-delà de 2-3 arguments j'ai du mal à lire un appel de fonction sans paramètres nommés.